cli/build.rs

```diff
@@ -70,13 +70,13 @@ fn main() {
   let bundle_path = o.join("COMPILER_SNAPSHOT.js");
   let snapshot_path = o.join("COMPILER_SNAPSHOT.bin");
   let mut custom_libs: HashMap<String, PathBuf> = HashMap::new();
+  custom_libs
+    .insert("lib.window.d.ts".to_string(), c.join("js/lib.window.d.ts"));
+  custom_libs
+    .insert("lib.worker.d.ts".to_string(), c.join("js/lib.worker.d.ts"));
   custom_libs.insert(
-    "lib.deno_main.d.ts".to_string(),
-    c.join("js/lib.deno_main.d.ts"),
-  );
-  custom_libs.insert(
-    "lib.deno_worker.d.ts".to_string(),
-    c.join("js/lib.deno_worker.d.ts"),
+    "lib.window_or_worker_global_scope.d.ts".to_string(),
```

Comment:
this feels like a really silly name...  `lib.deno_globals.d.ts`?

Reply:
Actually it's not my invention: https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope

Reply:
It's very explicit and descriptive thought, so it would really like to keep it, even if it's silly

Reply:
BAH... You hit me with prior art... Still sometimes we do silly things in standards (e.g. `.getElementById()`)... Just a thought.